install_config/configuring_authentication.adoc

```diff
@@ -1147,11 +1147,16 @@ by default *_/etc/ansible/hosts_*.
 
 *Configuring Apache*
 
-Unlike OpenShift Enterprise 2, this proxy does not need to reside on the same
+This proxy does not need to reside on the same
 host as the master. It uses a client certificate to connect to the master, which
 is configured to trust the `X-Remote-User` header.
 
-Configure Apache per the following:
+. Create the certificate for the Apache configuration. The certificate that you
+specify as the `SSLProxyMachineCertificateFile` parameter value is the proxy's
+client cert that is used to authenticate the proxy to the server. It must use
+`TLS Web Client Authentication` as the extended key type.
```

Comment:
So, you're creating the cert so that you can use it in the next step when configuring Apache? Would it be better to say this? Also, I have no idea how to create a cert. Would the reader?

Reply:
1. Yes.
2. I thought that "Create the certificate for the Apache configuration" ... "Configure Apache..." in the steps tied them together enough.
3. The user in the bug didn't seem to have a problem with creating the certs. @simo5, in the tickets you've seen, do the admins have trouble generating certs for auth?